packages/liquid-html-parser/src/stage-1-cst.ts

```diff
@@ -105,6 +107,21 @@ export interface ConcreteBasicNode<T> {
   locEnd: number;
 }
 
+export interface ConcreteLiquidDocParamNode
+  extends ConcreteBasicNode<ConcreteNodeTypes.LiquidDocParamNode> {
+  name: string;
+  value: string;
+  paramName: ConcreteTextNode;
+  paramDescription: ConcreteLiquidDocParamDescription;
+  paramType: ConcreteTextNode;
```

Comment:
Isn't this optional? Shouldn't this be ConcreteTextNode | null for when `{}` is not in the input string?